demo/icons/icon.html

```diff
@@ -49,25 +42,25 @@ <h3>Custom non-SVG (src attribute)</h3>
 
 		<h3>Color Override</h3>
 		<d2l-demo-snippet>
-			<div class="color-override">
+			<d2l-icon-demo-color-override>
 				<d2l-icon icon="d2l-tier3:calendar"></d2l-icon>
 				<d2l-icon src="../../components/icons/images/tier3/calendar.svg" size="tier3"></d2l-icon>
-			</div>
+			</d2l-icon-demo-color-override>
 		</d2l-demo-snippet>
 
 		<h3>Size Override</h3>
 		<d2l-demo-snippet>
-			<div class="size-override">
+			<d2l-icon-demo-size-override>
 				<d2l-icon icon="d2l-tier3:calendar"></d2l-icon>
 				<d2l-icon src="../../components/icons/images/tier3/calendar.svg" size="tier3"></d2l-icon>
 				<d2l-icon src="calendar-tier3.png" size="tier3"></d2l-icon>
-			</div>
+			</d2l-icon-demo-size-override>
 		</d2l-demo-snippet>
 
 		<h3>RTL</h3>
```

Comment:
It seems like the RTL specific snippet is somewhat redundant now.  Perhaps we just choose an icon for the basic icon snippet that happens to be RTL sensitive?

Reply:
Yeah I was thinking about that as well. Either that or just rely on the future visual diffs to test this.

Reply:
Updated to use a different icon that's RTL and removed this section.